pkg/apis/policy/v1alpha1/types_certificaterequestpolicy.go

```diff
@@ -249,9 +249,10 @@ type ValidationRule struct {
 	// `cr` (map) variable to the CEL expression containing `namespace` and
 	// `name` of the `CertificateRequest` resource.
 	//
-	// Examples:
-	// - Rule for namespaced DNSNames:
-	//   {"rule": "self.endsWith(cr.namespace + '.svc.cluster.local'"}
+	// Example (rule for namespaced DNSNames):
+	// ```
+	// rule: self.endsWith(cr.namespace + '.svc.cluster.local'
+	// ```
```

Comment:
question: Is this actually valid CEL syntax? I've not really played with it much but it looks wrong, I'd expect the following:

```suggestion
	// Example (rule for namespaced DNSNames):
	// ```
	// rule: self.endsWith(cr.namespace + '.svc.cluster.local')
	// ```
```

Reply:
🙈 Thanks, Ash!